docs/docs/constraints.md

```diff
@@ -30,20 +30,21 @@ Regex is allowed for LIKE and UNLIKE operators; to match ANY value, use the stri
 
 ### CLUSTER operator
 **Value** (optional): A string value.
-When a value is specified then tasks are launched on agent nodes with an attribute whose value matches exactly.
+When a value is specified, tasks are launched on agent nodes with an attribute whose value matches exactly.
 
 `CLUSTER` allows you to run all of your app's tasks on agent nodes that share a certain attribute.
-This is useful for example if you have apps with special hardware needs, or if you want to run them on the same rack for low latency.
+This is useful, for example, if you have apps with special hardware needs, or if you want to run them on the same rack for low latency.
 
-A field of `"hostname"` tells Marathon that launched tasks of the app/pod have affinity for each other and should be launched together on the same agent:
-* When value is specified then tasks are launched on the agent whose hostname matches the value.
-* When value is empty or unspecified then the first instance is launched on **any** agent node and the remaining tasks are launched alongside it on the same agent.
+The <hostname> field tells Marathon that the launched tasks of the app or pod should be launched together on the same agent:
```

Comment:
I'm still confused by this syntax:
* `The <hostname> field...`
* `The <hostname> value...`

This reads like there's a `<hostname>` element has a field and value. Which is not the case. A constraint is composed of the following:
* `field`: is the subject being tested. `"hostname"` (literally), or else is a string that reflects an attribute name of a mesos agent
  - "a field of `hostname`" means "... when the field is specified as `'hostname'`"
  - I don't think I was consistent here when I rewrote parts of this doc .. and I should have been
* `operator`: the test being performed
* `value`: what the field is being tested with/against

I am open to suggestions w/ respect to better ways to write about this stuff. I just want to make sure we're on the same page first.